pinot-core/src/main/java/org/apache/pinot/core/data/manager/InstanceDataManager.java

```diff
@@ -132,4 +133,9 @@ void reloadAllSegments(String tableNameWithType)
    * Returns the Helix property store.
    */
   ZkHelixPropertyStore<ZNRecord> getPropertyStore();
+
+  /**
+   * Returns the segment uploader.
```

Comment:
Add more details to the javadoc: explain what a segment uploader does and how server can use it.

Reply:
Comments updated.